styles.css

```diff
@@ -288,6 +288,34 @@ main {
   border-bottom-color: #EFB7EC;
 }
 
+.timer-countdown {
+  font-size: 70px;
+  font-weight: 700;
+  font-family: sans-serif;
+  margin: 0;
+}
+
+.timer-text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.activity-description {
+  font-size: 20px;
+  margin: 0;
+  font-weight: 200;
+}
+
+.log-activity {
```

Comment:
We can adjust the font weight of the activity card at a later time, but thank you for the attempt.